src/export/event-exporter.service.js

```diff
@@ -36,12 +36,23 @@ function eventExporter(jobLogger, udbApi, EventExportJob) {
   ex.export = function (format, email, properties, perDay, customizations) {
     var queryString = ex.activeExport.query.queryString,
         selection = ex.activeExport.selection || [],
-        eventCount = ex.activeExport.eventCount;
+        eventCount = ex.activeExport.eventCount,
+        gaBrand = customizations.brand || '',
```

Comment:
**Brand** heeft niks te maken met Google dus de _ga_ prefix is overbodig. Deze bepaalt hoe je pdf export er uitziet, bijvoorbeeld _Vlieg_ logo bovenaan.